lib/engine/config/game/g_1817.rb

```diff
@@ -224,6 +226,7 @@ module G1817
             "H9"
           ],
           "count": 2,
+          "when": "owning_corp_or_turn",
```

Comment:
Shouldn't this be done in 1817NA?

Reply:
Yes. I'll submit a separate PR so this one doesn't get held up

Reply:
it'd be easier to do one just in case something needs pinning

Reply:
pushed